tjmonopix2/scans/scan_ext_trigger.py

```diff
@@ -85,9 +67,7 @@ def timed_out():
                     self.log.info('Scan was stopped due to keyboard interrupt')
 
         self.pbar.close()
-        if max_triggers:
```

Comment:
I assume we want to disable the tlu module in any case, so removing this if clause still applies when bringing back timeout stop criterium

Reply:
Indeed. TLU always needs to be enabled and running for external trigger scans regardless of the stop condition. Hence, always disabled at the end as well.